[Integrations] Replaced screenshots on the Docker integration page  : cloud-native-management/meshery/integrations/docker

UtkarshMishra12 opened the pull request:
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
[Integrations] Replaced screenshots on the Docker integration page  : cloud-native-management/meshery/integrations/docker

This PR fixes #3424 

**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit 5f2d633c699acd858bd36792515b761291fa426f at: https://635e86417525ae499fb76641--layer5.netlify.app

UtkarshMishra12 commented:
@leecalcote I have made the necessary changes, let me know your thoughts on this.

![image](https://user-images.githubusercontent.com/67385503/198883403-da0b99fd-df14-45dd-bad5-2cc69159b050.png)

debo19 commented:
@UtkarshMishra12 please discuss your PR on the websites call tomorrow. There are a couple of other integrations that require updates to feature images used.

UtkarshMishra12 commented:
Is it ok @leecalcote  ? Or should I use other text for this?
![image](https://user-images.githubusercontent.com/67385503/199017592-b5070d51-5546-4f86-9644-9e69bfffbb2d.png)

leecalcote commented:
Looking good, @UtkarshMishra12 👏. Nice work.

UtkarshMishra12 commented:
@leecalcote  I've just changed the text on my local machine and did not push the code to the repo( Conforming the text).
Now I have moved it to the branch.
May I raise another PR?

leecalcote commented:
@UtkarshMishra12 yes, indeed. ✅
